src/NpmPackageFileParser.fs

```diff
@@ -11,14 +11,21 @@ type PackageFile = {
     [<YamlField("devDependencies")>] DevDependencies : Map<string, string> option
 }
 
+type PackageNameAndVersion = {
+    [<YamlField("name")>] Name: string
+    [<YamlField("version")>] Version: string
+}
+
 module Json5 =
     [<CLIMutable>]
     type Json5PackageFile = {
+        Name: string
+        Version: string
```

Comment:
The JSON5 parser behaves weirdly when the type is just the name and version, and parsing would fail. Adding it into the same type with the dependencies allows the name and version to be parsed.